src/commonMain/kotlin/com/soywiz/korma/geom/Matrix3D.kt

```diff
@@ -325,6 +330,39 @@ class Matrix3D {
         )
     }
 
+    fun setToFrustum(left: Float, right: Float, bottom: Float, top: Float, zNear: Float, zFar: Float): Matrix3D {
```

Comment:
Consider the `setToOrtho`. It uses `(x0, y0), (x1, y1)` notation (left, top, right, bottom). I understand that it is typical to use the (x0, x1) and (y0, y1) notation. But to be consistent setOrtho should be updated too.

```
fun setToOrtho(left: Float, top: Float, right: Float, bottom: Float, near: Float = 0f, far: Float = 1f): Matrix3D {
```

Reply:
I used pretty much exactly what was in GLU, so is it OK if I update setOrtho to be the same

Reply:
ok, modified setToOrtho argument order